src/sst_iter.rs

```diff
@@ -70,6 +71,7 @@ impl<'a> SstIterator<'a> {
             max_fetch_tasks,
             blocks_to_fetch,
             false,
+            true,
```

Comment:
I think this should be parameterized as `cache_blocks` rather than hard coded. I worry hard coding might lead to accidental clobbering if we ever create SstIterators in the compactor.